### ISSUE TITLE
Run this action without internet access fails to connect to https://api.globadge.com 

### ISSUE THREAD

**Issue by rajbos** [runforesight/workflow-telemetry-action#46]
Hi, we are testing out this action inside our corporate network, running on disconnected runners. 

I see you generate the line charts [here](https://github.com/runforesight/workflow-telemetry-action/blob/6705383eabd01833acfe8412ec697384830e1455/src/statCollector.ts#L318) by connecting to https://api.globadge.com, which does not work for us. Is there a workaround for this?

**Comment by rajbos** [comment]
Tagging @rwxdash as most active contributor?

**Comment by serkan-ozal** [comment]
Hi @rajbos,

Unfortunately, collected metrics are only provided through charts (generated by https://api.globadge.com/) currently. 

I think, you need the metrics in text format (plain text, JSON, CSV, etc ..), right? 

If so, I think your issue can be resolved by implementing the issue https://github.com/catchpoint/workflow-telemetry-action/issues/29?

**Comment by rajbos** [comment]
Yes. Having an input to skip the chart output and just give text output would be great. It think #29 would indeed fix this for us.
